src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
You can't import scss file this way, firstly they need to be compiled using tools like Sass. You can run command sass styles/main.scss styles/main.css
This command will compile main.scss into main.css